treeherder/seta/update_job_priority.py

```diff
@@ -0,0 +1,281 @@
+'''This module is used to add new jobs to the job priority table.
+
+This will query the Treeherder runnable api based on the latest task ID from
+mozilla-inbound's TaskCluster index.
+
+Known bug:
+ * Only considering mozilla-inbound makes SETA act similarly in all repositories where it is
+   active. Right now, this works for integration repositories since they tend to have
+   the same set of jobs. Unfortunately, this is less than ideal if we want to make this
+   work for project repositories.
+'''
+import copy
+import datetime
+import json
+import logging
+import os
+import time
+
+import requests
+from redo import retry
+
+from treeherder.seta.common import (db_map,
+                                    unique_key)
+from treeherder.seta.models import (DEFAULT_LOW_PRIORITY,
+                                    DEFAULT_TIMEOUT,
+                                    JobPriority)
+from treeherder.seta.runnable_jobs import RunnableJobs
+
+LOG = logging.getLogger(__name__)
+
+
+class ManageJobPriorityTable():
```

Comment:
Let's reconsider whether we need a class here, or if we can just define a set of methods which perform the operations we need (and return the appropriate data). For an example of that pattern, see `treeherder/perf/alerts.py` which contains a set of methods for generating and operating on performance alerts.